app/models/pageflow/account_member_query.rb

```diff
@@ -0,0 +1,57 @@
+module Pageflow
+  class AccountMemberQuery < ApplicationQuery
+    class Scope < Scope
+      # Account whose members we scope
+      # @return {Pageflow::Account}
```

Comment:
In the YARD doc they use `[]` for types. This is inconsistent in other files as well, but let's try to be consistent.